apps/studio/src/pages/sites/[siteId]/settings/index.tsx

```diff
@@ -1,217 +1,20 @@
-import { useState } from "react"
-import {
-  Button,
-  Center,
-  chakra,
-  FormControl,
-  HStack,
-  Text,
-  VStack,
-} from "@chakra-ui/react"
-import {
-  FormErrorMessage,
-  Infobox,
-  Input,
-  Toggle,
-  useToast,
-} from "@opengovsg/design-system-react"
-import { ResourceType } from "~prisma/generated/generatedEnums"
+import { useRouter } from "next/router"
 import { z } from "zod"
 
-import { PermissionsBoundary } from "~/components/AuthWrappers"
-import { ISOMER_SUPPORT_EMAIL } from "~/constants/misc"
-import { BRIEF_TOAST_SETTINGS } from "~/constants/toast"
-import { UnsavedSettingModal } from "~/features/editing-experience/components/UnsavedSettingModal"
-import { useNavigationEffect } from "~/hooks/useNavigationEffect"
+import { FullscreenSpinner } from "~/components/FullscreenSpinner"
 import { useQueryParse } from "~/hooks/useQueryParse"
-import { useZodForm } from "~/lib/form"
 import { type NextPageWithLayout } from "~/lib/types"
-import { setNotificationSchema } from "~/schemas/site"
-import { SiteBasicLayout } from "~/templates/layouts/SiteBasicLayout"
-import { trpc } from "~/utils/trpc"
 
 const siteSettingsSchema = z.object({
   siteId: z.coerce.number(),
 })
 
 const SiteSettingsPage: NextPageWithLayout = () => {
-  const toast = useToast()
-  const trpcUtils = trpc.useUtils()
+  const router = useRouter()
   const { siteId } = useQueryParse(siteSettingsSchema)
+  void router.replace(`sites/${siteId}/settings/agency`)
```

Comment:
question: I can't remember what we discussed about the rollout plan, are we doing full rollout here?